app/directors-corner.tsx

```diff
@@ -1,5 +1,90 @@
-import WorkInProgress from '@/components/work-in-progress';
+import { DM_Sans, DM_Serif_Display } from 'next/font/google';
+import Image from 'next/image';
+
+import HorsesRunning from '@/components/horses-running';
+
+const dmSerifDisplay = DM_Serif_Display({
+  weight: '400',
+  subsets: ['latin'],
+  display: 'swap',
+});
+
+const dmSans = DM_Sans({
+  weight: '400',
+  subsets: ['latin'],
+  display: 'swap',
+});
 
 export default function DirectorsCorner() {
-  return <WorkInProgress />;
+  return (
+    <div className="pt-20 flex-row items-center w-full h-fit bg-white">
```

Comment:
Won't leave  this comment often, but please use semantic tags instead of div spans. Feel free to explore MDN if you have doubts on which tag to use in what place.

Reply:
Okay. I'll get it done by evening today.

Reply:
Pushed an update for this